virtual_rainforest/models/hydrology/hydrology_model.py

```diff
@@ -736,15 +770,18 @@ def setup_hydrology_input_current_timestep(
         num_days=days,
         seed=seed,
     )
-    output["subcanopy_temperature"] = (
-        data["air_temperature"].isel(layers=layer_roles.index("subcanopy"))
+    output["surface_temperature"] = (
+        data["air_temperature"].isel(layers=layer_roles.index("surface"))
     ).to_numpy()
-    output["subcanopy_humidity"] = (
-        data["relative_humidity"].isel(layers=layer_roles.index("subcanopy"))
+    output["surface_humidity"] = (
+        data["relative_humidity"].isel(layers=layer_roles.index("surface"))
     ).to_numpy()
-    output["subcanopy_pressure"] = (
+    output["surface_pressure"] = (
         data["atmospheric_pressure_ref"].isel(time_index=time_index).to_numpy()
     )
+    output["surface_wind_speed"] = (
+        data["wind_speed"].isel(layers=layer_roles.index("surface"))
+    ).to_numpy()
```

Comment:
I think it's worth converting repeated statements like this into a loop of some kind once they get to 3 or more?

```python
for out_var, in_var in (
    ("surface_temperature", "air_temperature"),
    ("surface_humidity", "relative_humidity"),
    ("surface_wind_speed", "wind_speed")
):
    output[out_var] = data[in_var].isel(layers=layer_roles.index("surface")).to_numpy()

output["surface_pressure"] = (
    data["atmospheric_pressure_ref"].isel(time_index=time_index).to_numpy()
)
```